Document direction prefix for androidtv_remote send_command (#45253)
home-assistant/home-assistant.github.io: source/_integrations/androidtv_remote.markdown

{% enddetails %}

### Sending key commands

Whether a command does something depends on your Android TV device, its firmware, and the app in the foreground. Some devices remap or ignore certain keys, so a command like `GUIDE` or `MENU` might do nothing or trigger a different action than you expect. The commands are sent to your device as is, so Home Assistant cannot change how your device reacts to them.

> Pass any of the keys above as `command` in the `remote.send_command` action.

Reviewer: For clarification:
```suggestion
Pass any of the most common commands in the list above as `command` in the `remote.send_command` action.
```

  entity_id: remote.living_room_tv
```

### Press and release as separate events

> When a button, controller, or frontend card already emits separate press and release events, you can forward each one to the TV as its own service call by prefixing the command with a direction. Each call sends exactly one event, so the release fires when your source event fires — not when a fixed `hold_secs` timer ends.

Reviewer: Replace em dash by comma:
```suggestion
When a button, controller, or frontend card already emits separate press and release events, you can forward each one to the TV as its own service call by prefixing the command with a direction. Each call sends exactly one event, so the release fires when your source event fires, not when a fixed `hold_secs` timer ends.
```

  entity_id: remote.living_room_tv
```

### Launching apps and activities

> Use `remote.turn_on` with the `activity` field to launch an app or open a specific deep link. The value can be either an application ID (see the [Launching apps section](#launching-apps)) or a URL or scheme supported by the target app.

Reviewer: Since we are presenting more than 2 options:
```suggestion
Use `remote.turn_on` with the `activity` field to launch an app or open a specific deep link. The value can be an application ID (see the [Launching apps section](#launching-apps)), a URL, or a scheme supported by the target app.
```
